extensions/partners.rb

```diff
@@ -14,12 +14,6 @@ def initialize(app, options_hash={}, &block)
         @paginated_model_entry = :name
```

Comment:
Meta: does this entire file go away?

Reply:
I don't think it can be removed yet? The old index page still pulls from the partners space for a section so until the swapover happens there I wouldn't say it's possible to remove everything partners related in this repo.

Reply:
That makes sense then.